pkg/controller/gcp.go

```diff
@@ -43,7 +43,9 @@ func Setup(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter) error {
 		compute.SetupGlobalAddress,
 		compute.SetupNetwork,
 		compute.SetupSubnetwork,
+		compute.SetupFirewall,
 		container.SetupCluster,
+		container.SetupGKECluster,
```

Comment:
I believe this may have been inadvertently added during rebase.

Reply:
Oh, I think I messed up while rebasing the master to fix the conflicts, I will fix this up, and I think ideally it should just be `container.SetupCluster`. My bad, I will fix this up.